ui-lib/src/main/java/co/electriccoin/zcash/ui/MainActivity.kt

```diff
@@ -1,10 +1,13 @@
+@file:Suppress("DEPRECATION")
```

Comment:
Can we use the suppression locally so we do not apply it to the entire file?

Reply:
for some reason works for file only :(